yellow-paper/docs/rollup-circuits/index.md

```diff
@@ -7,18 +7,25 @@ title: Rollup Circuits
 Together with the [validating light node](../l1-smart-contracts/index.md) the rollup circuits must ensure that incoming blocks are valid, that state is progressed correctly and that anyone can rebuild the state.
 
 To support this, we construct a single proof for the entire block, which is then verified by the validating light node. 
-This single proof is constructed by recursively merging proofs together in a binary tree structure. 
+This single proof is constructed by recursively merging proofs together in two tree subtrees, and then combine these use those trees at the root . 
```

Comment:
```suggestion
This single proof is constructed by recursively merging proofs together in two tree subtrees, and then at last combine the final 2 subtrees in a final root circuit.
```

Reply:
I will update this. Take-away is that we build two trees that we are then using both outputs of in the root, so the "root" is kinda merging two proofs together, but the two proofs are not just the transaction ones (2 merges)

Also. Would like to remove the "rollup" from the circuit name if we can find something better as it seems misleading if it will end up as a validium depending on DA concerns.